core/embed/rust/src/ui/model_tr/component/button_controller.rs

```diff
@@ -117,6 +123,8 @@ where
         Self {
             pos,
             button_type: ButtonType::from_button_details(pos, btn_details),
+            pressed_since: None,
+            long_press_ms: 1000,
```

Comment:
Constant pls?

Reply:
Done in [8d9c7cb](https://github.com/trezor/trezor-firmware/pull/3290/commits/8d9c7cbe5b182269857a71fb39862f6aca9004e5)